include/envoy/http/codec.h

```diff
@@ -359,8 +361,10 @@ class Connection {
   /**
    * Dispatch incoming connection data.
    * @param data supplies the data to dispatch. The codec will drain as many bytes as it processes.
+   * @return Envoy::Http::Status indicating the status of the codec. Holds any errors found in the
+   * callbacks.
    */
-  virtual void dispatch(Buffer::Instance& data) PURE;
+  virtual Envoy::Http::Status dispatch(Buffer::Instance& data) PURE;
```

Comment:
You do not need to fully qualify Status namespace here. I think just `Status` should work.

Reply:
Done -- I changed the rest of the namespaces as well, opting for `Http::Status`, since `Http::Connection` is also used